general/clients/index.md

```diff
@@ -10,7 +10,7 @@ Clients connect your devices to your Jellyfin server and let you view your conte
 > [!NOTE]
 > If you are interested in helping out, please see our [contribution guide](xref:contrib-index) and feel free to contact us for more information!
 >
-> They aren't on this page, some of them can be found at the [jellyfin-archive organization on GitHub](https://github.com/jellyfin-archive).
+> If it isn't on this page, some clients can be found at the [jellyfin-archive organization on GitHub](https://github.com/jellyfin-archive).
```

Comment:
The subject here is `clients` which would require the use of `if they aren't` at the beginning of the sentence to indicate plurality.

Reply:
Fixed.